pmoves/services/hi-rag-gateway-v2/embeddings.py

```
@@ -0,0 +1,282 @@
+    """
+    import config as _cfg
+    with _embedder_lock:
+        old_model = _cfg.EMBEDDING_MODEL
+        old_type = _cfg.EMBEDDING_MODEL_TYPE
+        # Unload current model to free GPU memory
+        if _cfg._embedder is not None:
+            try:
+                if hasattr(_cfg._embedder, "model") and hasattr(_cfg._embedder.model, "to"):
+                    _cfg._embedder.model.to("meta")  # free GPU memory
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)